src/main/java/org/sonarlint/intellij/core/ProjectBindingManager.java

```diff
@@ -47,6 +56,36 @@ public ProjectBindingManager(Project project, SonarLintEngineManager engineManag
     this.console = console;
   }
 
+  @Override
+  public void initComponent() {
+    JobScheduler.getScheduler().scheduleWithFixedDelay(new Runnable() {
```

Comment:
Maybe it's better to extract this to a outer class which would be a project component (activating itself in the job scheduler), and have the  classes needed injected.